Revit_Core_Engine/Create/View/View3D.cs

```diff
@@ -56,6 +57,7 @@ public static View View3D(this Document document, string viewName = null, Boundi
                 try
                 {
                     (result as View3D).SetSectionBox(boundingBoxXyz);
+                    AlignCropBoxToBoundingBox(result, boundingBoxXyz);
```

Comment:
Is it always needed? Or only when there is a cropbox? Will it not crash the existing code?

Reply:
It is always needed and cropbox has to be active to get the correct result. Without alignment 3DView looks like this:

![image](https://github.com/BHoM/Revit_Toolkit/assets/71321938/7074f7d7-4980-4c18-9345-2da9602a0097)